src/components/AgreementSections/FormInfo.js

```diff
@@ -48,75 +57,123 @@ export default class FormInfo extends React.Component {
       newState.isPerpetualValues = [{ value: null, label: '' }, ...data.isPerpetualValues];
     }
 
+    if (data.reasonForClosureValues.length + 1 !== state.reasonForClosureValues.length) {
+      newState.reasonForClosureValues = [{ value: null, label: '' }, ...data.reasonForClosureValues];
+    }
+
+    // Check if status is closed and, if so, change state to allow reasonForClosure field
+    if (values.agreementStatus === 'closed') {
```

Comment:
Can you pop an entry for `closed` into `ui-agreements/src/constants/statuses.js` and use that?

Reply:
This has now been done.